docs/api-reference/orthographic-viewport.md

```diff
@@ -1,3 +1,48 @@
-# OrthographicViewport Class (Deprecated)
+# OrthographicViewport Class
```

Comment:
no change in content for the documents, just moved them back from deprecation.
will rewrite them once the APIs are fixed.